### PR TITLE
added a mailto anchor for all email texts - feature/21

### PR DESCRIPTION
**Pull request by StefanDiana**

### CODE REVIEW

File: sass/_mixins.scss [hunk 1]
```diff
@@ -54,3 +53,11 @@
...
   text-shadow: 5px 5px 2px black;
 }
+
+//Styles for hover span with <a> tag
+@mixin hover_link {
+  text-decoration: none;
+  font-weight: bold;
+  color: $primary_color;
+  transition: all 0.3s ease;
+}
```
Comment (emanuelstefan) [inline sass/_mixins.scss:63]: Awesome. GG! :)

**Review by emanuelstefan — APPROVED** [review]
Where is the .sass-cache folder? :)

**Comment by StefanDiana** [comment]
> Where is the .sass-cache folder? :)

image

File: sass/_mixins.scss (hunk 1 above)
Comment (StefanDiana) [reply sass/_mixins.scss:63]: GG????? Thanks :)

**Comment by emanuelstefan** [comment]
> > Where is the .sass-cache folder? :)
> 
> image

:))))))

File: sass/_mixins.scss (hunk 1 above)
Comment (emanuelstefan) [reply sass/_mixins.scss:63]: GG or GJ stand for Good game! / Good Job! :)
The first one is usually used in the gaming area/teams.

**Pull request merged**